### PR TITLE
DM-33922: use alembic

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: alembic/env.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+# my_important_option = config.get_main_option("my_important_option")
+# ... etc.
+
+
+def run_migrations_offline():
+    """Run migrations in 'offline' mode.
+
+    This configures the context with just a URL
+    and not an Engine, though an Engine is acceptable
+    here as well.  By skipping the Engine creation
```
Comment (wvreeven) [inline alembic/env.py:39]: `well.  By` -> `well. By`

File: alembic/env.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        target_metadata=target_metadata,
+        literal_binds=True,
+        dialect_opts={"paramstyle": "named"},
+    )
+
+    with context.begin_transaction():
+        context.run_migrations()
+
+
+def do_run_migrations(connection):
```
Comment (wvreeven) [inline alembic/env.py:58]: Please add a docstring for this function.

**Review by wvreeven — APPROVED** [review]
Looks good to me. Please review my few comments. I don't like the mixed cases of the initial characters of the comments in `alembic/env.py` so please change that if you agree. Do the missing types in the docstrings of function arguments (e.g. in `src/exposurelog/routers/find_exposures.py`) mean that sphinx now recognises the types from the type annotations or is this a cenvention in SQRE?

File: alembic/env.py (hunk 2 above)
Comment (r-owen) [reply alembic/env.py:58]: Good idea.

Note that 99% of this file was auto-generated by Alembic, including all the functions and their doc strings. So I'm not entirely sure what this one does, but I came up with something.

I also changed all comments to start with uppercase (the original file is an odd mix).

**Pull request merged**